### PR TITLE
Sanlouise add sections myva

### PR DESCRIPTION
**Pull request by sanlouise**
## Description
We need to add 'Claims and appeals', 'Healthcare' and 'Apply for Benefits' sections to the myVA2 so that we can start developing the components that need to be rendered within these sections.

## Testing done
Works well locally.

## Screenshots
image

## Acceptance criteria
- [x] Add 3 sections to myVA2

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/dashboard-2/components/Dashboard.jsx [hunk 1]
```diff
@@ -1,7 +1,17 @@
 import React from 'react';
 
-function Dashboard() {
-  return <h2>Dashboard Two!</h2>;
-}
+import ApplyForBenefits from './apply-for-benefits/ApplyForBenefits';
```
Comment (erikphansen) [inline src/applications/personalization/dashboard-2/components/Dashboard.jsx:3]: For now I'd opt to just keep these new components directly in the `dashboard-2/components` dir and if that doesn't scale we can change it. What do you think?

File: src/applications/personalization/dashboard-2/components/Dashboard.jsx [hunk 2]
```diff
@@ -1,7 +1,17 @@
...
+import ClaimsAndAppeals from './claims-and-appeals/ClaimsAndAppeals';
+import HealthCare from './health-care/HealthCare';
+
+const Dashboard = () => {
+  return (
+    <div className="vads-l-grid-container vads-u-padding-x--0">
```
Comment (erikphansen) [inline src/applications/personalization/dashboard-2/components/Dashboard.jsx:9]: remind me where the `My VA` h1 comes from 😆

**Review by erikphansen — APPROVED** [review]
Maybe just simplify the directory structure, but LGTM

File: src/applications/personalization/dashboard-2/components/Dashboard.jsx (hunk 1 above)
Comment (sanlouise) [reply src/applications/personalization/dashboard-2/components/Dashboard.jsx:3]: I see why you'd first want to keep things flat (usually I am all for that! 💯 ), but looking at the designs we're going to have quite a few section-specific components that I'd like to group together. If you are not too opinionated I'd vote to keep this structure in place, if you are alright with that?

File: src/applications/personalization/dashboard-2/components/Dashboard.jsx (hunk 2 above)
Comment (sanlouise) [reply src/applications/personalization/dashboard-2/components/Dashboard.jsx:9]: I've added the h1 in [this draft PR](https://github.com/department-of-veterans-affairs/vets-website/pull/15806) which I will open up for review after I merge this one 😄

File: src/applications/personalization/dashboard-2/components/Dashboard.jsx (hunk 1 above)
Comment (erikphansen) [reply src/applications/personalization/dashboard-2/components/Dashboard.jsx:3]: Sounds good!

File: src/applications/personalization/dashboard-2/components/Dashboard.jsx (hunk 2 above)
Comment (erikphansen) [reply src/applications/personalization/dashboard-2/components/Dashboard.jsx:9]: That works. Thanks.

**Pull request merged**